go/client/cmd_show_notifications.go

```diff
@@ -99,8 +99,16 @@ func (d *notificationDisplay) LoggedOut(_ context.Context) error {
 func (d *notificationDisplay) LoggedIn(_ context.Context, un string) error {
 	return d.printf("Logged in as %q\n", un)
 }
-func (d *notificationDisplay) ClientOutOfDate(_ context.Context, arg keybase1.ClientOutOfDateArg) error {
-	return d.printf("Client out of date, upgrade to \"%s\", uri \"%s\"\n", arg.UpgradeTo, arg.UpgradeURI)
+func (d *notificationDisplay) ClientOutOfDate(_ context.Context, arg keybase1.ClientOutOfDateArg) (err error) {
+	if arg.UpgradeMsg != "" {
+		err = d.printf("%s\n", arg.UpgradeMsg)
+	}
+	if arg.UpgradeTo != "" && arg.UpgradeURI != "" {
```

Comment:
Maybe an `||` here instead of `&&`, so that we don't confuse ourselves in the future by somehow forgetting to set both. Since this command is only ever used for debugging, you might also use `%v` instead of `%s` and drop the surrounding quotes.